src/GitHubExtension/Widgets/GitHubReviewWidget.cs

```diff
@@ -1,219 +1,63 @@
+    {
+        if (actionInvokedArgs.Verb == "Submit")
         {
-            { "openCount", 0 },
-            { "items", new JsonArray() },
-            { "referredName", ReferredName },
-            { "titleIconUrl", IconLoader.GetIconAsBase64("pulls.png") },
-            { "is_loading_data", true },
-        };
-        ContentData = issuesData.ToJsonString();
-    }
-
-    public void LoadContentData(IEnumerable<Octokit.Issue> items)
-    {
-        Log.Logger()?.ReportDebug(Name, ShortId, "Getting Data for Mentioned in Widget");
-
-        try
-        {
-            using var dataManager = GitHubDataManager.CreateInstance();
-
-            var issuesData = new JsonObject();
-            var issuesArray = new JsonArray();
-            issuesData.Add("openCount", items.Count());
-            foreach (var item in items)
-            {
-                var issue = new JsonObject
-                {
-                    { "title", item.Title },
-                    { "url", item.HtmlUrl },
-                    { "number", item.Number },
-                    { "date", TimeSpanHelper.DateTimeOffsetToDisplayString(item.UpdatedAt, Log.Logger()) },
-                    { "user", item.User.Login },
-                    { "avatar", item.User.AvatarUrl },
-                    { "iconUrl", IconLoader.GetIconAsBase64("pulls.png") },
-                };
-
-                var issueLabels = new JsonArray();
-                foreach (var label in item.Labels)
-                {
-                    var issueLabel = new JsonObject
-                    {
-                        { "name", label.Name },
-                        { "color", label.Color },
-                    };
-
-                    ((IList<JsonNode?>)issueLabels).Add(issueLabel);
-                }
-
-                issue.Add("labels", issueLabels);
-
-                ((IList<JsonNode?>)issuesArray).Add(issue);
-
-                var parsedUrl = item.HtmlUrl.Split('/');
-                var repo = parsedUrl[3] + '/' + parsedUrl[4];
-                issue.Add("repo", repo);
-            }
-
-            issuesData.Add("items", issuesArray);
-            issuesData.Add("referredName", ReferredName);
-            issuesData.Add("titleIconUrl", IconLoader.GetIconAsBase64("pulls.png"));
-
-            LastUpdated = DateTime.Now;
-            ContentData = issuesData.ToJsonString();
-            DataState = WidgetDataState.Okay;
-        }
-        catch (Exception e)
-        {
-            Log.Logger()?.ReportError(Name, ShortId, "Error retrieving data.", e);
-            DataState = WidgetDataState.Failed;
-            return;
-        }
-    }
+            var data = actionInvokedArgs.Data;
+            var dataObject = JsonNode.Parse(data);
+
+            if (dataObject == null)
+            {
+                return;
+            }
+
+            DeveloperLoginId = dataObject["account"]?.GetValue<string>() ?? string.Empty;

```

Comment:
Nit: It's a good idea to always use "DeveloperId" as the word for readability unless there is an actual difference in the variable (at least on the Extension repository).

Reply:
I would like to change some of the variable names too in this project. Any suggestions welcome.

Reply:
Agree, but recommend a separate PR for variable renames for anything not newly introduced with this PR.